### PR TITLE
Fedor Larionov docker-practice

### PR DESCRIPTION
**Pull request by ischade**

### CODE REVIEW

File: ops-tools/docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    redis:
+        image: redis:6.0.9-alpine
+        restart: unless-stopped
+        ports:
+            - "6379:6379"
+        networks:
+            - docker-practice-net
+
+    postgresql:
+        image: postgres:14
```
Comment (tatiana-vakhrameeva) [inline ops-tools/docker/docker-compose.yml]: Let's use alpine images - they are lightweight and should meet our needs.

File: ops-tools/docker/images/app/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM maven:3.6.3-jdk-11
```
Comment (tatiana-vakhrameeva) [inline ops-tools/docker/images/app/Dockerfile]: The same here - we can use alpine image

File: ops-tools/docker/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+services:
+    app:
+        build:
+            context: ../../
+            dockerfile: ./ops-tools/docker/images/app/Dockerfile
+        command: sh -c "mvn package && java -jar target/docker-practice-java-0.0.1-SNAPSHOT.jar"
+        volumes:
+            - ../../:/app
+        ports:
+            - "8080:8080"
```
Comment (tatiana-vakhrameeva) [inline ops-tools/docker/docker-compose.yml:11]: Why do we need this mapping?

**Review by tatiana-vakhrameeva — COMMENTED** [review]
Reviewed you code - looks good in general, but left a couple of comments of what should be fixed. Please take a look and let me know if you have any questions. Thanks!

File: ops-tools/docker/docker-compose.yml (hunk 3 above)
Comment (ischade) [reply ops-tools/docker/docker-compose.yml:11]: did it to accomplish the following task:
"При выполнении всех условий приложение должно успешно запуститься и по http
get запросу по url http://localhost:**8080/**api/check/postgres и
http://localhost:**8080**/api/check/redis должны получить ответ что сервис подключен"

File: ops-tools/docker/docker-compose.yml (hunk 1 above)
Comment (ischade) [reply ops-tools/docker/docker-compose.yml]: postgres and mvn images now are alpine-versions;

File: ops-tools/docker/docker-compose.yml (hunk 3 above)
Comment (tatiana-vakhrameeva) [reply ops-tools/docker/docker-compose.yml:11]: You talk about first part of task - it should be enough to run in docker postgres and redis. App should run without docker in the first task. It's in docker only for the second task. So no need to keep this line   - "8080:8080"

**Review by tatiana-vakhrameeva — COMMENTED** [review]
Looks good to me now. So task is completed. Thanks for the work!

**Pull request closed without merging**